Products/ZenUI3/browser/resources/js/zenoss/form/graphPanel.js

```diff
@@ -488,21 +493,20 @@
             config.hasMenu = false;
             config.xtype = 'europagraph';
             config.height = window.outerHeight * 0.75;
-            config.width = Math.min(window.outerWidth * 0.80, config.height * 1.6180339887);
-            config.maxWidth = 2000;
             config.autoScroll = true;
             delete config.html;
 
             var win = Ext.create('Zenoss.dialog.BaseWindow', {
                 cls: 'white-background-panel',
                 layout: 'fit',
-                width: config.width * 1.15,
-                height: config.height * 1.05,
+                // ZEN-30103
+                width: '90%',
```

Comment:
rather than 90% could we use calc(100% - 40px) or something similar?  Also, apostrophes not needed around value.

Reply:
extjs width config apply or percentage value according parent dimensions or number value, we can't use there css calc functionality.

Reply:
but we can calculate -40px of available screen width